src/client/telemetry/TelemetryEventNames.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+export enum telemetryEventNames {
```

Comment:
why a new constant file altogether. Check how other eventNames are logged of desktop extension

Reply:
Other eventNames in desktop extension are mostly being logged as a string. In copilot/studio interface we do have eventNames as constants - which is more streamlined. Therefore, this constant. 
This file has been added for the evenNames constant enum - can you recommend a more suitable place to keep this enum for desktop extension?

Reply:
Discussed on teams call - resolving comment